parsons/google/google_cloud_storage.py

```diff
@@ -439,12 +437,11 @@ def copy_bucket_to_gcs(
 
         # Create the transfer job
         create_result = client.create_transfer_job(create_transfer_job_request)
-        logger.info(f"Created TransferJob: {create_result.name}")
 
         polling = True
         wait_time = 0
         wait_between_attempts_in_sec = 10
-        max_wait_in_sec = 60 * 10  # Ten Minutes
+        # max_wait_in_sec = 60 * 10  # Ten Minutes
```

Comment:
If this variable isn't being used in the final version of the code, we should probably delete the line.